Calibration/PCCAlCaRecoProducers/python/AlcaPCCProducer_cfi.py

```diff
@@ -0,0 +1,9 @@
+import FWCore.ParameterSet.Config as cms 
+
+alcaPCCProducer = cms.EDProducer("AlcaPCCProducer",
```

Comment:
cfi file name should better match with the module instance defined in it: alcaPCCProducer_cfi.py